addons/sale/models/mail_thread.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class MailThread(models.AbstractModel):
+    _inherit = 'mail.thread'
+
+    def _track_finalize(self):
```

Comment:
I would probably move this override on "sale.order" model, as normally method is called on the model. No need to override the global (abstract) method.